eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -1615,14 +1615,17 @@ public function loadVersions(ContentInfo $contentInfo)
      */
     public function copyContent(ContentInfo $contentInfo, LocationCreateStruct $destinationLocationCreateStruct, APIVersionInfo $versionInfo = null)
     {
-        if (!$this->repository->canUser('content', 'create', $contentInfo, $destinationLocationCreateStruct)) {
```

Comment:
hmm, this implies that the logic in some limitations for location create struct are starting to be unnecessary then.

See also other api points using content create permissions.